core/domain/question_services_test.py

```diff
@@ -262,3 +262,6 @@ def test_get_question_batch(self):
         question_batch = question_services.get_questions_batch(
             coll_id_0, [skill_id], self.owner_id, 1)
         self.assertEqual(question_batch[0].title, question.title)
+        with self.assertRaises(Exception):
```

Comment:
assertRaisesRegexp and do some verification of the error message.

Reply:
Actually, this is not required as get_collection_by_id raises an exception if the collection id is not present.